Makefile

```diff
@@ -264,6 +264,9 @@ $(KUBECTL): ## Build kubectl
 	ln -sf "$(KUBECTL)" "$(TOOLS_BIN_DIR)/$(KUBECTL_BIN)"
 	chmod +x "$(TOOLS_BIN_DIR)/$(KUBECTL_BIN)" "$(KUBECTL)"
 
+.PHONY: $(KUBECTL_BIN)
+$(KUBECTL_BIN): $(KUBECTL) ## Building kubectl from the tools folder
```

Comment:
for this to work as you're using it below, we need to change KUBECTL_BIN on line 97 to `$(TOOLS_BIN_DIR)/kubectl` and then every place in the makefile that uses `$(TOOLS_BIN_DIR)/$(KUBECTL_BIN)` to just `$(KUBECTL_BIN)`

so that $(KUBECTL_BIN) becomes `/hack/tools/bin/kubectl`